.github/workflows/lint.yml

```diff
@@ -4,6 +4,7 @@ on:
   push:
     branches:
       - main
+      - user/dfriend01/70-hyperlinks-action
```

Comment:
This is duplicate with the pull request trigger. The reason I did it this way is that before creating a PR you can push your branch without running CI. Once you are ready to run CI you can create a PR

Reply:
Line 7 is now removed. Thank you.